tap_linkedin_ads/schemas/campaigns.json

```diff
@@ -259,7 +259,7 @@
 			"additionalProperties": false,
 			"properties": {
 				"amount": {
-					"type": ["null", "string"]
+					"type": ["null", "number"]
```

Comment:
Same comment as above. This also likely applies to any other `"number"` schema types.